README.md

```diff
@@ -1,76 +1,81 @@
 <h3 align="center">
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin for <a href="https://github.com/vendetta-mod/Vendetta">Vendetta</a>
+	Catppuccin for <a href="https://github.com/pyoncord/Bunny">Bunny</a> & <a href="https://github.com/enmity-mod/enmity">Enmity</a>
```

Comment:
I think we should still mention Vendetta in this list.

Reply:
The name of the repository will still be `vendetta` so I'd personally prefer if you could include it within this list. We can mention that Vendetta is EOL later on in the README